src/components/landing/HotQuestionSection.tsx

```
@@ -0,0 +1,120 @@
+  const [questions, setQuestions] = useState<QuestionPost[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+
+  // 탭 변경에 따른 데이터 로드
+  useEffect(() => {
+    const fetchQuestions = async () => {
+      setLoading(true);
+      try {
+        if (activeTab === "주간") {
+          const response = await questionPostApi.getWeeklyPopularQuestionPost();
+          if (response && response.questionPostsPage && response.questionPostsPage.content) {
+            setQuestions(response.questionPostsPage.content);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)